splinepy/helpme/integrate.py

```
@@ -160,34 +160,46 @@ def volume(spline, orders=None):
         raise NotImplementedError("integration only works for splines")
 
     # Retrieve aux info
     meas = _get_integral_measure(spline)
     positions, weights = _get_quadrature_information(spline, orders)
 
+    # define function to integrate
+    def _function(position):
+        if physical:
+            return function(spline.evaluate(position))
+        else:
+            return function(position)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)